contracts/azorius/BaseQuorumPercent.sol

```diff
@@ -51,4 +51,12 @@ abstract contract BaseQuorumPercent is OwnableUpgradeable {
     function meetsQuorum(uint256 _totalSupply, uint256 _yesVotes, uint256 _abstainVotes) public view returns (bool) {
         return _yesVotes + _abstainVotes >= (_totalSupply * quorumNumerator) / QUORUM_DENOMINATOR;
     }
+
+    /**
+     * Calculates the total number of votes required for a proposal to meet quorum.
+     * 
+     * @param _proposalId The ID of the proposal to get quorum votes for
+     * @return uint256 The quantity of votes required to meet quorum
+     */
+    function quorumVotes(uint32 _proposalId) public view virtual returns (uint256);
```

Comment:
@tbwebb22 should we make this external?

Reply:
hmm we could, wondering though if other strategies may have a need to call it internally

Reply:
yeah true, good call